contracts/StaticMarket.sol

```diff
@@ -0,0 +1,100 @@
+/*
+
+  << Static Market contract >>
+
+*/
+
+pragma solidity 0.7.5;
+
+import "./static/StaticUtil.sol";
+
+/**
+ * @title StaticMarket
+ * @author Wyvern Protocol Developers
+ */
+contract StaticMarket is StaticUtil {
+
+    string public constant name = "Static Market";
+
+	constructor (address atomicizerAddress)
```

Comment:
Spacing is different between here and two lines above for some reason?

Reply:
Right, tab snuck in there. Will fix.